### PR TITLE
fix: add custom topic to filter select box

### PR DESCRIPTION
**Pull request by brandonroberts**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recordings

image

## Steps to QA

1. Navigate to {{developerPreviewUrl}}/angular/dashboard/filter/recent
2. Note the custom topic in the selectbox

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by jpmcb — APPROVED** [review]

File: components/molecules/FilterCardSelect/filter-card-select.tsx [hunk 1]
```diff
@@ -58,6 +60,16 @@ const FilterCardSelect: React.FC<FilterCardSelectProps> = ({
         <SelectValue placeholder="select topic" />
       </SelectTrigger>
       <SelectContent className="bg-white">
+        {customTopic && (
```
Comment (nickytonline) [inline components/molecules/FilterCardSelect/filter-card-select.tsx:63]: Just wondering how we guard about putting anything here. If I add `potato`, it still renders in the list.

CleanShot 2024-02-13 at 08 10 32

I know these routes need some fixing, but just wondering if it makes more sense to add it to the list directly in https://github.com/open-sauced/app/blob/beta/lib/utils/getInterestOptions.ts ?

Comment (brandonroberts) [reply components/molecules/FilterCardSelect/filter-card-select.tsx:63]: Sure, we can add that one to the list, but we can't add all the common ones. That's we why allow custom ones to be done through the URL as a bit of a hidden feature.

Comment (nickytonline) [reply components/molecules/FilterCardSelect/filter-card-select.tsx:63]: I guess we can longterm validate it, so we don't have potato. 🥔

**Review by nickytonline — APPROVED** [review]
:shipit:

**Pull request merged**